ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/po/provision/cucushift-installer-rehearse-aws-ipi-proxy-po-provision-chain.yaml

```diff
@@ -0,0 +1,24 @@
+chain:
```

Comment:
@JoelSpeed 
add @jianzhangbjz review it because the original workflow is done by him.